build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
   id 'org.gradle.java-library'
 
   id 'com.github.ben-manes.versions' version '0.38.0'
+  id 'com.github.ManifestClasspath' version '0.1.0-RELEASE'
```

Comment:
Let's try to get rid of this dependency. Remember that you modified the `LongPathsEnabled` setting in Windows Registry, probably this setting itself is enough?

Pls remove this line, then run `./gradlew clean` (to remove the files created by a recent `./gradlew` invocation), and then run `./gradlew buildPlugin` again. Let's see if it works without this `ManifestClasspath` plugin activated.

Reply:
If that succeeds (so, if `ManifestClasspath` turns out to be NOT needed), then pls add a subsection under `Prerequisites` in `docs/development-guide.md` documenting which Windows Registry Key needs to be changed.